tests/integration/deploy-operator.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+
+# Copyright 2022 Red Hat, Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Exit immediately for non zero status
+set -e
+# Check unset variables
+set -u
+# Print commands
+set -x
+
+install-operator-k8s() { # installs istio-operator on kubernetes
+    local ROOT
+    ROOT="$(git rev-parse --show-toplevel)"
+    local TAG
+    TAG="$(git rev-parse HEAD)"
+    local NS="${NS:-openshift-operators}"
+    # var name IMAGE is in kind provisioner
+    local OPERATOR_IMAGE="${OPERATOR_IMAGE:-localhost:5000/istio-operator-integ}:${TAG}"
+    local ISTIO_REPO="${ISTIO_REPO:-quay.io/maistra-dev}"
+    local ISTIO_TAG="${ISTIO_TAG:-latest}"
```

Comment:
That's a local istio operator image istio-operator-integ.  I build it in this new job by `make image` and then push to  
localhost:5000/istio-operator-integ:${TAG} .
By default TAG will be the last commit SHA. If that's not the case, it will use `latest` as a tag.
This is not running maistra/istio build nor istio integ test.

Reply:
ah ok, so your SMCP deploys whatever is the latest OSSM tag? Please use maistra, not ossm as this is community work.

Reply:
I just updated the PR and using the CR example from deploy/examples . 
Yes, SMCP deploys whatever is the latest tags. I got those tags from marko's script.